### PR TITLE
chore(vue): Update vue-slider-component

### PR DESCRIPTION
**Pull request by mohdashraf010897**
**PR Type** `Bug Fix`

**Description** 
- https://github.com/appbaseio/reactivesearch/issues/1815
   - updated `vue-slider-component` to **v3.2.15**.
   - Demo 👉 https://www.loom.com/share/867f7300455348b6b24c9a9344628868
   - Demo With SSR 👉 https://www.loom.com/share/01c3650665134332840bd460979580b9

[📔  Notion](https://www.notion.so/appbase/ReactiveSearch-issues-86750916b182477793caefc4d89b377f)

### CODE REVIEW

File: packages/vue/src/components/range/RangeInput.jsx [hunk 1]
```diff
@@ -113,7 +113,7 @@ const RangeInput = {
 			}
 		},
 		handleOnChange(value) {
-			this.handleChange(value, 'change');
+			this.handleChange(value ?? this.$props.range, 'change');
```
Comment (bietkul) [inline packages/vue/src/components/range/RangeInput.jsx]: @mohdashraf010897 Avoid using `Nullish` operator, it is not supported by some browsers.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

File: packages/vue/src/components/range/RangeSlider.jsx [hunk 2]
```diff
@@ -200,11 +198,8 @@ const RangeSlider = {
 								height={4}
 								enable-cross={false}
 								{...{ props: this.$props.sliderOptions }}
-								{...{
-									on: {
-										input: this.handleSlider,
-									},
-								}}
+								onDrag-end={this.handleSlider}
+								tooltip="always"
```
Comment (bietkul) [inline packages/vue/src/components/range/RangeSlider.jsx:201]: @mohdashraf010897 Move it to above line no 200 `sliderOptions` prop can have priority.

File: packages/vue/src/components/range/DynamicRangeSlider.jsx [hunk 3]
```diff
@@ -314,6 +316,7 @@ const DynamicRangeSlider = {
 							height={4}
 							enable-cross={false}
 							{...{ props: this.$props.sliderOptions }}
+							tooltip="always"
```
Comment (bietkul) [inline packages/vue/src/components/range/DynamicRangeSlider.jsx:318]: @mohdashraf010897 Check range slider comment

**Review by bietkul — CHANGES_REQUESTED** [review]

File: packages/vue/src/components/range/RangeInput.jsx (hunk 1 above)
Comment (mohdashraf010897) [reply packages/vue/src/components/range/RangeInput.jsx]: fixed issues https://github.com/appbaseio/reactivesearch/pull/1853/commits/6a1e2a831f23dbf88a79eed5f676751d1909daea

File: packages/vue/src/components/range/RangeSlider.jsx (hunk 2 above)
Comment (mohdashraf010897) [reply packages/vue/src/components/range/RangeSlider.jsx:201]: fixed here https://github.com/appbaseio/reactivesearch/pull/1853/commits/6a1e2a831f23dbf88a79eed5f676751d1909daea

File: packages/vue/src/components/range/DynamicRangeSlider.jsx (hunk 3 above)
Comment (mohdashraf010897) [reply packages/vue/src/components/range/DynamicRangeSlider.jsx:318]: fixed here https://github.com/appbaseio/reactivesearch/pull/1853/commits/6a1e2a831f23dbf88a79eed5f676751d1909daea

**Review by bietkul — APPROVED** [review]

**Pull request merged**